### DISCUSSION TITLE
Clarification on Logging Practices and Privacy in OpenFGA

### DISCUSSION THREAD

**Discussion by naveengovindarajan** [openfga/community#308] [Q&A] [answered]
Hello OpenFGA Community,

We are considering using OpenFGA for our project and have a question about its logging capabilities. 

1. Are these logs shared automatically with the OpenFGA development team or to their servers? I could not find any relevant info on their documentation regarding the same.
2. If so, is there a way to opt out or configure the privacy settings to prevent automatic log sharing, considering the open-source nature of the tool?

Thanks in advance and apologies if my question was too basic.

**Comment by aaguiarz** [comment, marked as answer]
Hi @naveengovindarajan 

Thanks for considering OpenFGA! 

Logs are not shared.

Let us know if we can help in any other way.

**Reply by naveengovindarajan** [reply]
Thanks @aaguiarz  for the prompt answer. Much appreciated.
